src/schemas/clinical-resolvers/clearClinicalSubmissionResolver.ts

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) 2023 The Ontario Institute for Cancer Research. All rights reserved
+ *
+ * This program and the accompanying materials are made available under the terms of
+ * the GNU Affero General Public License v3.0. You should have received a copy of the
+ * GNU Affero General Public License along with this program.
+ *  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND ANY
+ * EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES
+ * OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT
+ * SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT,
+ * INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED
+ * TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS;
+ * OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER
+ * IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+ * ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+import { GlobalGqlContext } from '../../app';
+import submissionAPI from '../../submission/submission-api';
+import { convertClinicalSubmissionDataToGql } from '../utils';
+
+const clearClinicalSubmissionResolver = {
+  clearClinicalSubmission: async (
+    obj: unknown,
+    args: { programShortName: string; fileType: string; version: string },
+    contextValue: any,
+  ) => {
+    const { programShortName, fileType, version } = args;
+    const response = await submissionAPI.clearFileDataFromActiveSubmission(
+      programShortName,
+      fileType,
+      version,
+      (<GlobalGqlContext>contextValue).egoToken,
```

Comment:
For consistency w/ other functions I would change line 28 to `contextValue: GlobalGqlContext,` and 
line 35 to `contextValue.egoToken,`

Reply:
unfortunately, the _buildSubgraphSchema_ method does not accept resolvers of custom context type. Hence the typecast.

Reply:
OK I see now, sorry VSCode TS version was not set correctly